src/tests/Interop/MarshalAPI/IUnknown/IUnknownTest.cs

```diff
@@ -138,69 +82,31 @@ public void GetObjectForIUnknownTest()
             {
                 ptr = Marshal.GetIUnknownForObject(obj);
 
-                Object tmpObj = Marshal.GetObjectForIUnknown(ptr);
+                var tmpObj = Marshal.GetObjectForIUnknown(ptr);
 
                 //compare the new object reference with the original object, they should point to the same underlying object
                 if (!object.ReferenceEquals(obj, tmpObj))
+                {
                     throw new Exception("GetObjectForIUnknown returned a different object. Original: " + obj + ", New: " + tmpObj);
+                }
             }
             finally
             {
                 if (ptr != IntPtr.Zero)
+                {
                     Marshal.Release(ptr);
+                }
             }
         }
     }
 
-    public void GetUniqueObjectForIUnknownTest()
```

Comment:
Are the removed cases based on them already being in libraries? And I guess it looks like they weren't called here either.

Reply:
Correct.